sdk/appplatform/azure-mgmt-appplatform/azure/mgmt/appplatform/_app_platform_management_client.py

```diff
@@ -12,60 +12,238 @@
 from msrest.service_client import SDKClient
 from msrest import Serializer, Deserializer
 
+from azure.profiles import KnownProfiles, ProfileDefinition
+from azure.profiles.multiapiclient import MultiApiClientMixin
 from ._configuration import AppPlatformManagementClientConfiguration
-from .operations import ServicesOperations
-from .operations import AppsOperations
-from .operations import BindingsOperations
-from .operations import DeploymentsOperations
-from .operations import Operations
-from . import models
 
 
-class AppPlatformManagementClient(SDKClient):
+
+class AppPlatformManagementClient(MultiApiClientMixin, SDKClient):
     """REST API for Azure Spring Cloud
 
+    This ready contains multiple API versions, to help you deal with all Azure clouds
+    (Azure Stack, Azure Government, Azure China, etc.).
+    By default, uses latest API version available on public Azure.
+    For production, you should stick a particular api-version and/or profile.
+    The profile sets a mapping between the operation group and an API version.
+    The api-version parameter sets the default API version if the operation
+    group is not described in the profile.
+
     :ivar config: Configuration for client.
     :vartype config: AppPlatformManagementClientConfiguration
 
-    :ivar services: Services operations
-    :vartype services: azure.mgmt.appplatform.operations.ServicesOperations
-    :ivar apps: Apps operations
-    :vartype apps: azure.mgmt.appplatform.operations.AppsOperations
-    :ivar bindings: Bindings operations
-    :vartype bindings: azure.mgmt.appplatform.operations.BindingsOperations
-    :ivar deployments: Deployments operations
-    :vartype deployments: azure.mgmt.appplatform.operations.DeploymentsOperations
-    :ivar operations: Operations operations
-    :vartype operations: azure.mgmt.appplatform.operations.Operations
-
     :param credentials: Credentials needed for the client to connect to Azure.
     :type credentials: :mod:`A msrestazure Credentials
      object<msrestazure.azure_active_directory>`
-    :param subscription_id: Gets subscription ID which uniquely identify the
+    :param subscription_id: Subscription credentials which uniquely identify
      Microsoft Azure subscription. The subscription ID forms part of the URI
      for every service call.
     :type subscription_id: str
+    :param str api_version: API version to use if no profile is provided, or if
+     missing in profile.
     :param str base_url: Service URL
+    :param profile: A profile definition, from KnownProfiles to dict.
+    :type profile: azure.profiles.KnownProfiles
     """
 
-    def __init__(
-            self, credentials, subscription_id, base_url=None):
+    DEFAULT_API_VERSION = '2020-07-01'
+    _PROFILE_TAG = "azure.mgmt.appplatform.AppPlatformManagementClient"
+    LATEST_PROFILE = ProfileDefinition({
+        _PROFILE_TAG: {
+            None: DEFAULT_API_VERSION,
+            'sku': '2019-05-01-preview',
```

Comment:
The `2020-07-01` also contains `sku`, why there is such an entry?

Reply:
For 2019-05-01-preview, the resource name is sku, but for 2020-07, the resource name is skus. Because default api is 2020-07 which does not has sku, so it is listed here

Reply:
Thanks for explaining